src/backend/server.js

```diff
@@ -131,6 +132,55 @@ app.post('/log', wrap(async (req, res) => {
   res.send('')
 }))
 
+app.post('/password_reset', wrap(async (req, res) => {
+  // All errors in this endpoint will result in a 200 response.
+  // This prevents people from mining emails through brute
+  // force.
+  if(!req.body.email) {
+    return res.status(200).end();
+  }
+
+  let user = await models.User.findOne({ where: {email:req.body.email} });
+
+  if(!user) {
+    return res.status(200).end();
+  }
+
+  console.log(req.body.email)
+  console.log(JSON.stringify(user))
+
+  // Generate a globally unique reset token for the user
+  // We create it as a property of the user since we only
+  // issue one at a time, and require that the token
+  // match the email address when updating.
+  await user.set('resetToken',uuid.v4());
```

Comment:
I don't think this requires an await, but not sure.  I've usually just been doing user.resetToken = uuid.v4().  Does .set do something different?